build/elasticsearch/bin/es-docker

```diff
@@ -36,4 +35,28 @@ done < <(env)
 # will run in.
 export ES_JAVA_OPTS="-Des.cgroups.hierarchy.override=/ $ES_JAVA_OPTS"
 
+# Determine if x-pack platinum is enabled
```

Comment:
Doesn't this test just determine if x-pack is installed, rather to what level it is licenced?